java/example/src/main/java/com/youtube/vitess/example/VitessClientExample.java

```diff
@@ -38,37 +39,39 @@ public static void main(String[] args) throws Exception {
 
     // Connect to vtgate.
     HostAndPort hostAndPort = HostAndPort.fromString(args[0]);
+    InetSocketAddress addr =
+        new InetSocketAddress(hostAndPort.getHostText(), hostAndPort.getPort());
     Context ctx = Context.getDefault().withDeadlineAfter(Duration.millis(5 * 1000));
-    RpcClient client = new GrpcClientFactory().create(
-        ctx, new InetSocketAddress(hostAndPort.getHostText(), hostAndPort.getPort()));
-    VTGateConn conn = new VTGateConn(client);
-
-    String keyspace = "test_keyspace";
-    Iterable<String> shards = Arrays.asList("0");
-    Map<String, Object> bindVars =
-        new ImmutableMap.Builder<String, Object>().put("msg", "V is for speed").build();
+    try (RpcClient client = new GrpcClientFactory().create(ctx, addr);
+        VTGateConn conn = new VTGateConn(client);) {
```

Comment:
Is it recommended style to include the unnecessary trailing semicolon? Examples in general Java docs seem to omit it. Same for the Cursor below.